lib/booktype/skeleton/dev_settings.py.original

```diff
@@ -3,10 +3,9 @@ import os
 from .base import *
 
 # WEB SITE URL
-THIS_BOOKTYPE_SERVER = os.environ.get('BOOKTYPE_SERVER', '')
+THIS_BOOKTYPE_SERVER = os.environ.get('BOOKTYPE_SERVER', '127.0.0.1:8000')
```

Comment:
Do we need a similar change for prod_settings.py.original please?

Reply:
Hi @danielhjames , no, we don't need it. I've made it in dev settings to make it easier to kickstart dev instance from scratch. For production instance developer will change settings by himself.

Reply:
Sounds good!